auxprogs/joingenes/Makefile

```
@@ -1,20 +1,20 @@
 CC=g++
-CFLAGS=-Wall -std=gnu++0x
+CXXFLAGS+=-Wall -std=gnu++0x
 
 all: joingenes
 
 joingenes: joingenes.o jg_transcript.o jg_ios.o jg_transcript.h jg_ios.h
-	$(CC) $(CFLAGS) $(CXXFLAGS) $(CPPFLAGS) $(LDFLAGS) joingenes.o jg_transcript.o jg_ios.o -o joingenes
+	$(CC) $(CPPFLAGS) $(CXXFLAGS) joingenes.o jg_transcript.o jg_ios.o -o joingenes $(LDFLAGS)
 	cp joingenes ../../bin/
 
 joingenes.o: joingenes.cpp jg_transcript.h jg_ios.h
-	$(CC) -c $(CFLAGS) $(CXXFLAGS) $(CPPFLAGS) ${LDFLAGS} joingenes.cpp
+	$(CC) -c $(CPPFLAGS) $(CXXFLAGS) joingenes.cpp
```

```suggestion
	$(CXX) -c $(CXXFLAGS) $(CPPFLAGS) ${LDFLAGS} joingenes.cpp
```
- replace CC by CXX as mentioned before
- why was ${LDFLAGS} removed?
- why has flag order changed?

---

CXX correct, LDFLAGS can be removed